benchmark_runner/benchmark_operator/benchmark_operator_workloads.py

```diff
@@ -224,7 +235,44 @@ def system_metrics_collector(self, workload: str):
         # verify that data upload to elastic search
         if self.__es_host:
             self.__es_operations.verify_es_data_uploaded(index=f'system-metrics',
-                                                         uuid=self.__oc.get_long_uuid(workload=workload))
+                                                         uuid=self.__oc.get_long_uuid(workload=workload), fast_check=True)
+
+    def get_metadata(self, kind: str = None, database: str = None):
+        """
+        This method return run metadata
+        @param kind: pod or vm
```

Comment:
What is the default here (and should there even be a default?  And obviously I'll have to change that in the kata PR).